worker/download.go

```diff
@@ -611,8 +611,8 @@ func (s *slabDownload) overdrive(ctx context.Context, resps *sectorResponses) (r
 			case <-ctx.Done():
 				return
 			case <-timer.C:
-				if canOverdrive(timeout()) {
-					for {
+				for {
+					if canOverdrive(timeout()) {
```

Comment:
Felt right to move this inside since every launch of a job affects the outcome of `canOverdrive`.

Reply:
Yup, makes sense I think